packages/flask-aserto/src/flask_aserto/_defaults.py

```diff
@@ -1,20 +1,37 @@
 import re
-from typing import Callable
+from dataclasses import dataclass
+from typing import Callable, TypeVar, Union
 
 from aserto.client import Identity, ResourceContext
 from flask import request
+from flask.wrappers import Response
 
 __all__ = [
     "create_default_policy_path_resolver",
     "DEFAULT_RESOURCE_CONTEXT_PROVIDER_FOR_ENDPOINT",
     "DEFAULT_RESOURCE_CONTEXT_PROVIDER_FOR_DISPLAY_STATE_MAP",
 ]
 
+@dataclass
+class Obj:
+    id: str
+    objType: str
+
+
 IdentityMapper = Callable[[], Identity]
 StringMapper = Callable[[], str]
+ObjectMapper = Callable[[], Obj]
 ResourceMapper = Callable[[], ResourceContext]
 DEFAULT_DISPLAY_STATE_MAP_ENDPOINT = "/__displaystatemap"
 
+@dataclass(frozen=True)
+class AuthorizationError(Exception):
+    policy_instance_name: str
+    policy_path: str
+
+
+Handler = TypeVar("Handler", bound=Callable[..., Union[Response, Response]])
```

Comment:
This definition is strange what does `Union[Response, Response]` mean? It's the same as `Response`.
Furthermore, in `main` we define Handler as `Handler = TypeVar("Handler", bound=Callable[..., Any])` because a flask route can return things that aren't a `Response`. We return dicts which get converted to JSON, and we return ("message", error_code) tuples as well.

I wouldn't change the definition in this PR. That will cause `pyright` errors in the todo app.